target/linux/ramips/patches-5.4/991-at803x.patch

```diff
@@ -0,0 +1,141 @@
+From 60ae82b0ea56c279be384b99cd2a42ae5ba7c5c7 Mon Sep 17 00:00:00 2001
```

Comment:
Commit title's prefix should be "ramips:" if this is a platform specific patch.

Reply:
Ok, shall change that.